package.json

```diff
@@ -54,4 +54,4 @@
     "test:apex": "sh ./scripts/runLocalTests.sh",
     "test:lwc": "sfdx-lwc-jest --coverage --skipApiVersionCheck"
   }
-}
\ No newline at end of file
+}
```

Comment:
It's odd - I can't resolve this one. Genuinely no clue what it says is changed. I'm editing in the web interface, so it shouldn't show anything. Does it mess up your Windows dev environment?

Reply:
It's just noise on the PR. I can fix it afterwards.